nautobot_ssot/integrations/slurpit/jobs.py

```diff
@@ -0,0 +1,110 @@
+"""Slurpit DataSource job class."""
+
+import asyncio
+
+import slurpit
+from django.templatetags.static import static
+from django.urls import reverse
+from nautobot.extras.choices import SecretsGroupAccessTypeChoices, SecretsGroupSecretTypeChoices
+from nautobot.extras.jobs import BooleanVar, Job, ObjectVar
+from nautobot.extras.models import ExternalIntegration
+
+from nautobot_ssot.integrations.slurpit import constants
+from nautobot_ssot.integrations.slurpit.diffsync.adapters.nautobot import NautobotDiffSyncAdapter
+from nautobot_ssot.integrations.slurpit.diffsync.adapters.slurpit import SlurpitAdapter
+from nautobot_ssot.jobs.base import DataMapping, DataSource
+
+loop = asyncio.new_event_loop()
```

Comment:
What's the purpose of the async loop here? I saw you already did the async stuff in the Slurpit adapter.

Reply:
Removed

Reply:
I'm still seeing this in the code?

Reply:
I can see this has been removed.